app/src/main/res/values/strings.xml

```diff
@@ -4,136 +4,137 @@
 
     <!-- Common refresh toasts -->
     <string name="refresh_message_result">%1$d notification(s) received</string>
-    <string name="refresh_message_no_results">Everything is up-to-date</string>
-    <string name="refresh_message_error">%1$d subscription(s) could not be refreshed\n\n%2$s</string>
-    <string name="refresh_message_error_one">Subscription could not be refreshed: %1$s</string>
+    <string name="refresh_message_no_results">Everything is up to date</string>
+    <string name="refresh_message_error">Could not refresh %1$d subscription(s)\n\n%2$s</string>
+    <string name="refresh_message_error_one">Could not refresh subscription: %1$s</string>
 
     <!-- Main activity: Action bar -->
     <string name="main_action_bar_title">Subscribed topics</string>
-    <string name="main_menu_notifications_enabled">Notifications enabled</string>
-    <string name="main_menu_notifications_disabled_forever">Notifications disabled</string>
-    <string name="main_menu_notifications_disabled_until">Notifications disabled until %1$s</string>
+    <string name="main_menu_notifications_enabled">Notifications on</string>
+    <string name="main_menu_notifications_disabled_forever">Notifications off</string>
+    <string name="main_menu_notifications_disabled_until">Notifications off till %1$s</string>
     <string name="main_menu_settings_title">Settings</string>
     <string name="main_menu_report_bug_title">Report a bug</string>
     <string name="main_menu_report_bug_url">https://github.com/binwiederhier/ntfy/issues</string>
-    <string name="main_menu_docs_title">Read the docs</string>
+    <string name="main_menu_docs_title">Read the documentation</string>
     <string name="main_menu_docs_url">https://ntfy.sh/docs</string>
-    <string name="main_menu_rate_title">Rate this app ⭐</string>
+    <string name="main_menu_rate_title">Rate the app ⭐</string>
 
     <!-- Main activity: Action mode -->
     <string name="main_action_mode_menu_unsubscribe">Unsubscribe</string>
     <string name="main_action_mode_delete_dialog_message">
-        Do you really want to unsubscribe from the selected topic(s) and
-        permanently delete all the notifications you received?
+        Unsubscribe from the selected topic(s) and
+        delete all notifications you ever received for good?
     </string>
-    <string name="main_action_mode_delete_dialog_permanently_delete">Permanently delete</string>
+    <string name="main_action_mode_delete_dialog_permanently_delete">Delete for good</string>
     <string name="main_action_mode_delete_dialog_cancel">Cancel</string>
 
     <!-- Main activity: List and such -->
     <string name="main_item_status_text_one">%1$d notification</string>
     <string name="main_item_status_text_not_one">%1$d notifications</string>
-    <string name="main_item_status_reconnecting">reconnecting …</string>
+    <string name="main_item_status_reconnecting">Reconnecting …</string>
```

Comment:
```suggestion
    <string name="main_item_status_reconnecting">Reconnecting…</string>
```

Reply:
That fails in languages where the verb, object, subject falls in a different order.
It also fails when cases change depending on the actor/enactor of the reconnection.
It isn't obvious when translating that it is part of a sentence.
Would be better to have all sentences available in full, or in the very least a mention in the WL context.